source/_data/MCPubs.bib

```diff
@@ -306,6 +309,9 @@ @Article{terra:arxiv19
   biburl        = {https://dblp.org/rec/journals/corr/abs-1904-08480.bib},
   eprint        = {1904.08480},
   url           = {http://arxiv.org/abs/1904.08480},
+  publist_link = {paper || http://arxiv.org/abs/1904.08480},
```

Comment:
```suggestion
  publist_link = {paper || https://arxiv.org/abs/1904.08480},
```

Reply:
Please use https links

Reply:
Changed to https. Also changed other arxiv links to https.